example/ipsec/odp_ipsec.c

```diff
@@ -241,7 +241,7 @@ typedef odp_queue_t (*queue_create_func_t)
 typedef odp_event_t (*schedule_func_t) (odp_queue_t *);
 
 static queue_create_func_t queue_create;
-static schedule_func_t schedule;
+static schedule_func_t schedule_fn;
```

Comment:
This rename is irrelevant to `odp_crypto_op()/odp_crypto_op_enq()` return codes and should be split to a separate patch.

Reply:
Renames moved to separate patch in v3

Reply:
Thank you!

Reply:
I could add your reviewed-by tag and merge this PR. Have you reviewed the entire set, or only this patch?

Reply:
Yes, please. I have reviewed whole series.

Reviewed-by: Dmitry Eremin-Solenikov